app/assets/stylesheets/file.css

```diff
@@ -120,7 +120,7 @@
 
   .folder-title {
     display: flex;
-    align-items: flex-start;
+    align-items: center;
```

Comment:
Doesn't this mess with the alignment when the folder name wraps to two lines?

Reply:
I could be wrong but I don't think we can have both. If that is the expected behavior then all folder/files need to align with the top of the icon and not the middle.

Reply:
I think this problem comes from having the icons bigger than the text.  Can we add additional padding on the top of the text and keep `flex-start`?

Reply:
I wonder if using `margin-top: auto` and `margin-bottom: auto` on the folder and file text (instead of changing align-items) would help with the vertical centering?

Example
<img width="642" alt="margin-top-bottom" src="https://github.com/user-attachments/assets/e37a5b64-0b9e-4bed-af19-f764a89b2fc7" />

Example
<img width="570" alt="margin-top-bottom-2" src="https://github.com/user-attachments/assets/ecc2395a-e01a-4722-bb83-5ed3aa2e8097" />

Reply:
It sort of fixes things but it still looks weird. 
![Screenshot 2025-02-27 at 3 30 19 PM](https://github.com/user-attachments/assets/33025ddd-e0a2-4372-9517-87ef2142e6dc)

Reply:
I think I figured it out. 
![Screenshot 2025-02-27 at 3 46 54 PM](https://github.com/user-attachments/assets/f23d681f-fd01-46a8-afa3-b2be57d426f8)

Reply:
@jcoyne I have approval for the new view from @alundgard. This just needs a dev review.